.changeset/sharp-news-smoke.md

```diff
@@ -0,0 +1,26 @@
+---
+'@backstage/backend-common': minor
+---
+
+Added support for Azure AD authentication for PostgreSQL. To define Azure credentials add `tenantId`, `clientId`, `clientSecret` to `app-config.yaml` file.
+
+```
+backend:
+  database:
+    .....
+    auth:
```

Comment:
Any additions like this should be reflected [in the config schema](https://github.com/backstage/backstage/blob/dcd7e1622c3b8008f7561ba48f94010672159a48/packages/backend-common/config.d.ts#L71) too.

Reply:
I've added changes to the config schema file